locale/en-us.yaml

```
@@ -1701,3 +1701,7 @@ operation_success_local:
 err_initialize_runtime_event_handler:
   other: |
     Could not determine how to display progress of runtime installation.
+solver_err:
+  other: The Platform failed to resolve the dependencies for this build. {{.V0}}\n{{.V1}}
+transient_solver_tip:
+  other: You may want to retry this command, as it can lead to a different result. 
```

```suggestion
  other: You may want to retry this command, as it can lead to a different result. 
```

This goes against the promise of the platform. Why would retrying it give a different result? Same request should give the same result, we should not have error messages that indicate otherwise.

---

The solver has this error:  https://github.com/ActiveState/cli/blob/cd0ba72d833dcf6e3fafbb858c31144afc8b4fa8/pkg/platform/api/inventory/inventory_models/solver_error.go#L24-L25   I think it only happens when there is a resource problem that interrupted the solve.

Will check with PB.

---

It's whenever a resource failure occurred during the solve: https://activestate.slack.com/archives/CCELJM8GL/p1627406491195600

Should hardly ever be the case...

---

```suggestion
  other: You may want to retry this command if the failure was related to a resourcing or networking issue.
```

I'm not a big fan of this still, but I'm especially not a fan of "it may lead to different results". Technically that is accurate, but I don't think we should be setting expectations where retrying magically fixes things.